HelpSource/Classes/Buffer.schelp

```diff
@@ -18,9 +18,9 @@ subsection:: Buffer Numbers and Allocation
 
 Although the number of buffers on a server is set at the time it is booted, memory must still be allocated within the server app before they can hold values. (At boot time all buffers have a size of 0.)
 
```

Comment:
I agree that the best wording was

"Under normal usage this argument should be left as nil. Supplying a number bypasses the server's buffer allocator, which can lead to conflicts. See discussion of the allocator scheme here for more info."

Reply:
We don't currently have a discussion of the allocator scheme anywhere in the Help to point to. Adding this last sentence would be a mistake.

Reply:
yes, you are right about this last sentence. So in my perspective, the best would be:

"Under normal usage this argument should be left as nil. Supplying a number bypasses the server's buffer allocator, which can lead to conflicts."